src/Auth/UserIsAdminAuthValidator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace RebelCode\EddBookings\RestApi\Auth;
+
+use Dhii\Exception\CreateInvalidArgumentExceptionCapableTrait;
+use Dhii\I18n\StringTranslatingTrait;
+use Dhii\Util\Normalization\NormalizeIterableCapableTrait;
+use Dhii\Util\Normalization\NormalizeStringCapableTrait;
+use Dhii\Validation\CreateValidationFailedExceptionCapableTrait;
+use Dhii\Validation\ValidatorInterface;
+
+/**
+ * An authorization validator that validates whether a user is an administrator.
+ *
+ * @since [*next-version*]
+ */
+class UserIsAdminAuthValidator implements ValidatorInterface
+{
+    /* @since [*next-version*] */
+    use NormalizeStringCapableTrait;
+
+    /* @since [*next-version*] */
+    use NormalizeIterableCapableTrait;
+
+    /* @since [*next-version*] */
+    use CreateInvalidArgumentExceptionCapableTrait;
+
+    /* @since [*next-version*] */
+    use CreateValidationFailedExceptionCapableTrait;
+
+    /* @since [*next-version*] */
+    use StringTranslatingTrait;
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    public function validate($userId)
+    {
+        $errors = [];
+
+        if (!user_can($userId, 'manage_options')) {
+            $errors[] = 'User does not have admin capability "manage_options"';
```

Comment:
Why not i18n-ize error messages?

Reply:
Is it because those errors are intended to be i18n-ized closer to the view layer?

Reply:
No, missed it. Will i18n-ize.